api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1832,6 +1833,7 @@ def transfer_with_liquid_class(
         :param keep_last_tip: When ``True``, the pipette keeps the last tip used in the transfer attached. When
             ``False``, the last tip will be dropped or returned. If not set, behavior depends on the value of
             ``new_tip``. ``new_tip="never"`` keeps the tip, and all other values of ``new_tip`` drop or return the tip.
+        :param tip_racks: If provided, use the tip racks given instead of the pipette's assigned tip racks.
```

Comment:
Heh, I think you'll have to add some verbiage about which version this argument was added in.

Reply:
I would suggest this for all three methods:
```suggestion
        :param tip_racks: A list of tip racks to pick up from for this command. If not provided, the pipette will pick up from its associated :py:obj:`~.InstrumentContext.tip_racks`. Providing this argument does not change the value of ``tip_racks``.
        
        .. versionchanged:: 2.25 
            Added the ``tip_racks`` parameter.

```